src/sentry/api/endpoints/project_repo_path_parsing.py

```diff
@@ -119,7 +119,7 @@ def post(self, request: Request, project) -> Response:
 
         branch = installation.extract_branch_from_source_url(repo, source_url)
         source_path = installation.extract_source_path_from_source_url(repo, source_url)
-        stack_root, source_root = find_roots(stack_path, source_path)
+        stack_root, source_root = find_roots(FrameFilename({"filename": stack_path}), source_path)
```

Comment:
A `FrameFilename` has more information than `stack_path`, thus, I can use it within `find_roots`.